gateway-js/src/__tests__/executeQueryPlan.test.ts

```diff
@@ -20,7 +20,10 @@ import { ApolloGateway } from '..';
 import { ApolloServerBase as ApolloServer } from 'apollo-server-core';
 import { getFederatedTestingSchema } from './execution-utils';
 import { Schema, Operation, parseOperation, buildSchemaFromAST, arrayEquals } from '@apollo/federation-internals';
-import { addResolversToSchema, GraphQLResolverMap } from '../schema-helper';
+import {
+  addResolversToSchema,
+  GraphQLResolverMap,
+} from '@apollo/subgraph/dist/schema-helper';
```

Comment:
Hmm. Is this better than something like `../../../subgraph-js/src/schemaHelper`?

Reply:
Can't reference the file directly, but I can use `src` files instead
[29ae85c](https://github.com/apollographql/federation/pull/1757/commits/29ae85cf638df94d547c34417747ad0758206038)